org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/command/MarkAsResolvedCommand.java

```diff
@@ -55,10 +64,51 @@ private static Optional<ResolvedBinding> getBinding(IMarker marker) {
   }
 
   @Override
-  protected void execute(IMarker selectedMarker) {
-    var msg = new MessageBox(Display.getDefault().getActiveShell());
-    msg.setText("TODO");
-    msg.open();
+  protected void execute(IMarker selectedMarker, IWorkbenchWindow window) {
+    var checkJob = new Job("Check user permissions for administering issues") {
```

Comment:
Maybe `Check user permissions for setting the issue resolution` or something like this?